.github/scripts/test_trymerge.py

```diff
@@ -704,33 +605,33 @@ def test_get_classifications(self, *args: Any) -> None:
         )
         self.assertTrue(len(pending) == 0)
         self.assertTrue(len(failed) == 0)
-        self.assertTrue(len(ignorable["FLAKY"]) == 2)
-        self.assertTrue(len(ignorable["BROKEN_TRUNK"]) == 4)
+        self.assertTrue(len(ignorable["FLAKY"]) == 4)
```

Comment:
why did all of these swap?

Reply:
Because of the example we are using in this test I think, https://github.com/pytorch/pytorch/pull/109584 uses ghstack, so 2 of broken trunk failures are not there on the "true" base commit used by Dr.CI.  On the other hand, they are found elsewhere in trunk, so these 2 failures now belong to the flaky group instead.

If we take https://github.com/pytorch/pytorch/pull/109584 as the source of truth, then the swap is correct.